docs/algorithms/mopt/implementation_en.md

```diff
@@ -0,0 +1,16 @@
+# MOpt implementation in fuzzuf
+
+## Reference MOpt Implementation
+
+- Version: 2.52b
+- Commit: https://github.com/puppet-meteor/MOpt-AFL/tree/a9a5dc5c0c291c1cdb09b2b7b27d7cbf1db7ce7b
+
+## Differences from the original implementation
+
+Since original AFL-MOpt implements their method in a weird way, fuzzuf MOpt rewrote all algorithms to reproduce original MOpt-AFL behaviour with fuzzuf architecture. Basically there is no difference between MOpt-AFL and fuzzuf MOpt.
```

Comment:
I think "weird" is not an appropriate word to describe the situation. It should be replaced with a more concrete and moderate word.

Reply:
would be fixed in another comment